internal/controller/numaresourcesoperator_controller.go

```diff
@@ -672,6 +684,36 @@ func (r *NUMAResourcesOperatorReconciler) mcpToNUMAResourceOperator(ctx context.
 	return requests
 }
 
+func (r *NUMAResourcesOperatorReconciler) configMapToNUMAResourceOperator(ctx context.Context, cmObj client.Object) []reconcile.Request {
```

Comment:
why do we need this if `configMapPredicates` is already filtering the config maps based on the NROP label?

Reply:
`configMapToNUMAResourceOperator` is a function which converts the `ConfigMap` (A) events  to `NUMAResourceOperator` requests, because the `NUMAResourcesOperatorReconciler` expects to reconcile the NUMAResourceOperator CR and not a `ConfigMap` objects

The configMapPredicates is under filter layer that minimizes the `ConfigMap` events that gets into the `configMapToNUMAResourceOperator` function.

Reply:
ack I see how builder Watches is expecting an event handler and options, thanks for clarifying